main.py

```diff
@@ -61,11 +61,11 @@ def search_results():
     response = response.json()
     for i in range(12):
         try:
-            channel_id.append(response["items"][i]['snippet']['channelId'])
+            channelId.append(response["items"][i]['snippet']['channelId'])

```

Comment:
Tiny nit, but I like these new names - without the unnecessary '_' - a lot better than the old ones!